packages/esm-patient-banner-app/src/banner/patient-banner.component.tsx

```diff
@@ -27,15 +27,15 @@ interface PatientBannerProps {
 
 const PatientBanner: React.FC<PatientBannerProps> = ({
   patient,
-  patientUuid,
+  patientUuid
 }) => {
   const { t } = useTranslation();
   const { currentVisit } = useVisit(patientUuid);
   const [showContactDetails, setShowContactDetails] = useState(false);
   const [hasActiveVisit, setActiveVisit] = useState(false);
   const state = useMemo(() => ({ patientUuid }), [patientUuid]);
   const toggleContactDetails = useCallback(
-    () => setShowContactDetails((value) => !value),
+    () => setShowContactDetails(value => !value),
```

Comment:
Could you please check, that you have the newest version of eslint/tslint/typescript installed. The formatting changes seem to be a step backwards.

Reply:
@nmalyschkin  Actually when I committed and the pre-commit tests ran, the prettier package gave me errors in all these files, which I solved by running `npm run prettier`.

What should I do now to undo all the prettier changes? Should I close this one and create a new PR?